raiden/blockchain_events_handler.py

```diff
@@ -406,6 +409,27 @@ def handle_channel_settled(raiden: "RaidenService", event: Event):
         raiden.handle_and_track_state_change(channel_settled)
 
 
+def get_our_address_by_canonical_identifier(
+    chain_state: ChainState, canonical_identifier: CanonicalIdentifier, raiden: "RaidenService") -> Optional[AddressHex]:
+    token_network = get_token_network_by_identifier(
+        chain_state, TokenNetworkID(canonical_identifier.token_network_address)
+    )
+    for address, channels in token_network.channelidentifiers_to_channels.items():
+        if canonical_identifier.channel_identifier in channels \
+                and (raiden.address == address or raiden.get_light_client_transport(to_checksum_address(address))):
+            return address
+    return None
+
+
+def get_channelstate_by_canonical_identifier(
+    chain_state: ChainState,
+    canonical_identifier: CanonicalIdentifier,
+    raiden: "RaidenService") -> Optional[NettingChannelState]:
+    address = get_our_address_by_canonical_identifier(chain_state, canonical_identifier, raiden)
+    if address is None:
```

Comment:
This could be:
```python
if address:
     return views.get_channelstate_by_canonical_identifier_and_address(chain_state,canonical_identifier, address)
else:
     return None
```
Or you can also do that and remove the else since if you don't return anything it's the same as None

Reply:
I think [guard clauses](https://refactoring.com/catalog/replaceNestedConditionalWithGuardClauses.html) are cleaner than elses. They let you keep the error handling and exceptional situations at one place, and prevent nested ifs.

Also, I don't like the idea of using a "implicit" return. I agree in this situation with [Explicit is better than implicit](https://en.wikipedia.org/wiki/Zen_of_Python)

Reply:
I think using 
```python
if address is None:
    return None
```
It's redundant that's all, i mean i think you should improve at least that you know, this is the same as the example of using if something is not None